landlab/components/lithology/lithology.py

```diff
@@ -260,9 +279,60 @@ def __init__(self, grid, thicknesses, ids, attrs, layer_type="MaterialLayers"):
             except IndexError:
                 self.add_layer(self._init_thicknesses[i], self._layer_ids[i])
 
+        self.dz_advection = dz_advection
+        self.rock_id = rock_id
+
     def __getitem__(self, name):
         return self._get_surface_values(name)
 
+    @property
+    def dz_advection(self):
+        """Rate of vertical advection.
+
+        Parameters
+        ----------
+        dz_advection : float, `(n_nodes, ) shape array, or at-node field array optional
+            Change in rock elevation due to advection by some external process.
+            This can be changed using the property setter.
+
+        Returns
+        -------
+        current rate of vertical advection
+        """
+        return return_array_at_node(self._grid, self._dz_advection)
+
+    @dz_advection.setter
+    def dz_advection(self, dz_advection):
+        _ = return_array_at_node(
```

Comment:
`_ = ...`? Does this not cause the linter complain about an unused variable?

Reply:
Nope. The linter knows that `_` is fake. But I removed it anyway.